libs/shared/src/lib/components/markdown/components/markdown-editor/interfaces/markdown-button.interface.ts

```diff
@@ -0,0 +1,8 @@
+import { MarkdownActionEnum } from '../enums';
+import { MarkdownIconNameType } from '../types';
+
+export interface MarkdownButton {
+    readonly markdownSyntaxEnum: MarkdownActionEnum;
+    readonly matIconName: MarkdownIconNameType;
+    readonly getMarkdownSyntax: (markdownText: string) => string;
```

Comment:
```suggestion
    readonly getMarkdown(context?: string): string;
```

Цей метод на вхід приймає звичайний текст (context), а повертає текст огорнутий в синтаксис markdown.

Інколи текст на вхід може бути відсутнім (коли не виділено нічого в textarea).